introduction.md

```diff
@@ -0,0 +1 @@
+hello  helloasdfasdfasdfasfdasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasdfasfdasdfasdfasdfasdfasdfasfdasdfasdfasdfasdfasdfasdfa;sldkfjasd;flkjasdf;lerkjzd;flkjeqr;lzkdjad;fglkjert;ldksfjg;slfkj
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.